translations/fr/app.php

```diff
@@ -176,12 +169,8 @@
     'Verification Code' => 'Code de vérification',
     'View / Download' => 'Visualiser / Télécharger',
     'Was Generated By' => 'Généré Par',
-    'Wavelength' => 'Longueur d\'onde',
```

Comment:
This translation seems to be used:         
_form-characterize.php	
<?= Html::label(Yii::t('app', 'Wavelength') . ' (nm)  <font color="red">*</font>', 'wavelength') ?>      [position 420:40]